Makefile

```diff
@@ -46,14 +46,17 @@ endif
 #----------------------------------------------------------------------
 
 # Explicitly set the default target to do everything that isn’t already done
-default: | init assets all public ;
+default: | init lint assets all public ;
 
 # Run anything that needs doing post-checkout to make this buildable
 init: node_modules ;
 
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -type f -name '*.json' -print -exec jsonlint -q '{}' \;
```

Comment:
First of all, this shouldn't have `\;`, but `+`, second... `-exec` is more efficient than `-execdir` here, because it will cause only one invocation of `jsonlint`.